frontend/syntaxmeets/src/components/SyntaxChat/SyntaxChat.js

```diff
@@ -69,7 +74,22 @@ const SyntaxChat = (props) => {
     }
   };
 
+  function triggerPicker(event) {
+    event.preventDefault();
+    SetEmojiPicker(!emojiPickerState);
```

Comment:
💡 **Tip:** instead of directly toggling the state, it's a good practice to toggle the state using previous props.
```
setEmojiPicker((prevState) => (!prevState));
```